### ISSUE TITLE
script/run_msra_ner.sh: line 33: 17400 Killed

### ISSUE THREAD

**Issue by flytoylf** [PaddlePaddle/ERNIE#93]
Load pretraining parameters from model_path/params.
script/run_msra_ner.sh: line 33: 17400 Killed  
求问下这个是什么原因？

**Comment by nbcc** [comment]
cpu 还是 gpu 环境呢？

看是否内存/显存不足。

GPU 
https://github.com/PaddlePaddle/LARK/blob/68e04f38a59becb3d1baa6d127e8657e486e6415/ERNIE/run_sequence_labeling.py#L43

CPU
https://github.com/PaddlePaddle/LARK/blob/68e04f38a59becb3d1baa6d127e8657e486e6415/ERNIE/run_sequence_labeling.py#L46

**Comment by nightkid101** [comment]
> Load pretraining parameters from model_path/params.
> script/run_msra_ner.sh: line 33: 17400 Killed
> 求问下这个是什么原因？

您好，请问您这个问题解决了吗?我也遇到同样的问题。
@flytoylf 

`Device count: 4
Num train examples: 9600
Max train steps: 24000
Num warmup steps: 0
Theoretical memory usage in training: 2389.192 - 2502.964 MB
Load pretraining parameters from ERNIE_stable-1.0.1/params.
./script/run_ChnSentiCorp.sh: line 30: 13392 Killed                  python -u run_classifier.py --use_cuda false --verbose true --do_train true --do_val true --do_test true --batch_size 1 --init_pretraining_params ERNIE_stable-1.0.1/params --train_set task_data/chnsenticorp/train.tsv --dev_set task_data/chnsenticorp/dev.tsv --test_set task_data/chnsenticorp/test.tsv --vocab_path ERNIE_stable-1.0.1/vocab.txt --checkpoints ./checkpoints --save_steps 1000 --weight_decay 0.01 --warmup_proportion 0.0 --validation_steps 100 --epoch 10 --max_seq_len 256 --ernie_config_path config/ernie_config.json --learning_rate 5e-5 --skip_steps 10 --num_iteration_per_drop_scope 1 --num_labels 2 --random_seed 1
`
